pkg/pillar/types/zedroutertypes.go

```diff
@@ -1413,3 +1413,28 @@ type NestedAppDomainStatus struct {
 func (status NestedAppDomainStatus) Key() string {
 	return status.UUIDandVersion.UUID.String()
 }
+
+// FsUsedMetric - contains the used and allocated
+// space of a NestedApp running on a runtime
+// app instance.
+type FsUsedMetric struct {
+	UsedMb      float64
+	AllocatedMb float64
+}
+
+// NestedAppRuntimeDiskMetric - contains the overall
+// fs usage of a volume instance attached to a runtime
+// app instance.  All storage of child/NestedApps running in
+// this runtime app instance is included.
+type NestedAppRuntimeDiskMetric struct {
+	UUID             string
+	TotalMb          float64
+	UsedMb           float64
+	AllocatedMb      float64
+	DependentSpaceMb map[string]FsUsedMetric
```

Comment:
Can you add to the comment above what the string key is for this map? I'm guessing it is the UUID string of the nested app.